mercury/templates/dashboard.html

```diff
@@ -1,60 +1,10 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8" http-equiv="refresh" content="5; URL=/dashboard/"/>
-    <title>Dashboard</title>
-    {%  load static %}
-    <link rel="stylesheet" type="text/css" href="{% static 'mercury/style.css' %}">
-</head>
-<body>
+{% extends 'banner.html' %}
+
+{% block content %}
+<meta charset="UTF-8" http-equiv="refresh" content="5; URL=/dashboard/"/>
```

Comment:
Meta tags should always be in the `<head>` element, but because of the way the include is structured, this page has the `<meta>` in the `<body>`. Have we considered just making the banner be a small block that is included in other pages, rather than it containing the entire html, head, and body, elements that other pages use?

Reply:
Yes exactly its a violation. I thought about it but couldn't find another way of doing it. I will search for other possible ways today before merging or we can merge it since its working and make amendments later.

Reply:
My suggestion is to make the `banner.html` code just have elements about the banner (don't include html, head, or body tags) and use something like this in the other pages (near the top of the other pages, at the start of the body section).
```
{% block foo %}
   {% include "banner.html" %}
{% endblock %}
```

Reply:
That's a good suggestion, Thank you will read more about it and if it's the write thing to do in our case I will modify the code. I will label the pull request as Do not merge

Reply:
Modified the code so as to use include tags instead of extends